tests/covalent_tests/executor/executor_plugins/timing_plugin_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Agnostiq Inc.
```

Comment:
Will need to remove this file as well (and any other such files which make the timing plugin a part of covalent).

Reply:
So the game plan is for the functional tests PR to get merged in first and then this wouldn't show up in the diff